experimental/InjectiveResolutions/src/InjectiveResolutions.jl

```diff
@@ -0,0 +1,1098 @@
+module InjectiveResolutions
+using ..Oscar
+using ..Oscar: IntegerUnion # add other things that are not exported from Oscar here
+# functions with new methods
+import ..Oscar:
+  coefficient_ring,
+  gens,
+  ModuleGens,
+  SubModuleOfFreeModule,
+  is_zm_graded,
+grading_group,
+  elem_type,
+  singular_module, 
+  singular_generators,
+  cone,
+  faces,
+  hyperplanes,
+  is_pointed,
+  zonotope,
+  degree,
+  zero,
+  one,
+  inv,
+  degree,
+  _degree_fast,
+  dim,
+  is_subset,
+  coordinates,
+  intersect,
+  primitive_generator,
+  evaluate,
+  coefficients,
+  dim,
+  is_normal,
+  standard_basis,
+  _build_sparse_row,
+  normal_form,
+  lift_std,
+  sparse_row,
+  syzygy_module,
+  kernel,
+  annihilator,
+  twist,
+  _saturation,
+  free_resolution,
+  _reduce, 
+  singular_assure,
+  oscar_assure, 
+  _graded_kernel,
+  oscar_generators,
+  singular_poly_ring,
+  _simple_kernel,
+  _extend_free_resolution, 
+  free_show
+
+
+import ..Oscar.Singular: 
+  FreeModule, 
+  has_global_ordering,
+  svector,
+  Module
+ 
+
+
+for i in names(Oscar)
+  !isdefined(Oscar, i) && continue
+  @eval import Oscar: $i
+  #@eval export $i
+end
+
+for i in names(Oscar.Orderings)
+  !isdefined(Oscar.Orderings, i) && continue
+  @eval import Oscar.Orderings: $i
+  #@eval export $i
+end
+
+#=
+for i in names(Oscar; all=true)
+  !isdefined(Oscar, i) && continue
+  @eval import Hecke: $i
+end
+=#
```

Comment:
```suggestion

import ..Oscar.Singular: 
  FreeModule, 
  has_global_ordering,
  svector,
  Module
 


for i in names(Oscar)
  !isdefined(Oscar, i) && continue
  @eval import Oscar: $i
  #@eval export $i
end

for i in names(Oscar.Orderings)
  !isdefined(Oscar.Orderings, i) && continue
  @eval import Oscar.Orderings: $i
  #@eval export $i
end

#=
for i in names(Oscar; all=true)
  !isdefined(Oscar, i) && continue
  @eval import Hecke: $i
end
=#
```

Reply:
I don't understand this last bit. What's the difference between before and after?